hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientListenerServiceImpl.java

```diff
@@ -164,7 +164,11 @@ public void run() {
             final ClientResponse clientResponse = serializationService.toObject(packet.getData());
             final int callId = clientResponse.getCallId();
             final Data response = clientResponse.getResponse();
-            handleEvent(response, callId, conn);
+            try {
+                handleEvent(response, callId, conn);
+            } finally {
+                conn.decrementPacketCount();
```

Comment:
What does packet count mean? Number of pending packets? If so, why isn't it called 'decrementPendingPacketCount'? This way code becomes self documenting.